internal/node/hash.go

```diff
@@ -0,0 +1,153 @@
+package node
+
+import (
+	"fmt"
+	"hash"
+	"hash/fnv"
+	"sort"
+	"strings"
+)
+
+// MethodsResult contains all available rpc methods of node
+type MethodsResult struct {
+	Version int      `json:"version"`
+	Methods []string `json:"methods"`
+}
+
+// RPCMethodsResponse is rpc response od rpc_methods call
+type RPCMethodsResponse struct {
+	JSONRPC string        `json:"jsonrpc"`
+	Result  MethodsResult `json:"result"`
+	ID      int           `json:"id"`
+}
+
+// RPCChainTypeResponse is rpc response of system_chainType call
+type RPCChainTypeResponse struct {
+	JSONRPC string `json:"jsonrpc"`
```

Comment:
you might wanna add common properties to some generic type JsonRpcResponse, and reference it in every method response type